heron/statemgrs/src/java/com/twitter/heron/statemgr/zookeeper/curator/CuratorStateManager.java

```diff
@@ -77,8 +77,7 @@ public void initialize(Config newConfig) {
 
       String newConnectionString = tunneledResults.first;
       if (newConnectionString.isEmpty()) {
-        LOG.warning("Cannot connect to tunnelHost: " + tunnelConfig.getTunnelHost());
-        throw new IllegalArgumentException("Bad connectionString: " + connectionString);
+        throw new IllegalArgumentException("Cannot connect to tunnelHost: " + tunnelConfig.getTunnelHost() + "\nBad connectionString: " + connectionString);
```

Comment:
let's remove the newline from the exception message. This complicates log parsing when grepping or using loglens.